lib/admin/route/admin.instance.create.id/admin.instance.create.id.controller.js

```diff
@@ -148,6 +143,12 @@ instanceCreateIdController.validate = (pageInstance, pageData, POST) => {
       const schemaProperty = schema.properties[idSeed]
       let validationError = validator.validate(idSeedValue, schemaProperty).errors[0]
       if (validationError) {
+        validationError.values = {
```

Comment:
Ensure that error messages when creating ids for new instances get properly interpolated